### DISCUSSION TITLE
100K Users - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#562] [Announcements]
# 100K Users - Revolt

Breaking the 100K Milestone!

https://revolt.chat/posts/100K-users

**Comment by Nightcaat** [comment]
100,000 revolters and still no skill-based matchmaking

**Reply by Rexogamer** [reply]
skill issue

**Comment by SnenxyTengoku** [comment]
when one morbillion users

**Comment by ashleyirispuppy143** [comment]
ONE IN A KIRRILION

**Comment by lo2dev** [comment]
I'm about to morb

**Comment by sussycatgirl** [comment]
![deez](https://media.discordapp.net/attachments/966400443720826970/1118218450519736330/0beeb1f7-54b5-49d5-97d5-346a3fc38cce.png)

**Comment by katelyynn** [comment]
one million users ! !

**Reply by searinminecraft** [reply]
100 thousand*

**Reply by katelyynn** [reply]
one million clearly :trolleyzoom:

**Comment by DeclanChidlow** [comment]
Huzzah! Can't wait to see how this great platform prospers.

**Comment by ktwrd** [comment]
holy based, 100k nerds

**Comment by searinminecraft** [comment]
Even though that 100K is not that big as Discord's millions, it's still a pretty awesome number for this platform.

**Congratulations Revolt!!**

**Comment by FanMclaine** [comment]
#revoltsweep

**Comment by Andre601** [comment]
I think the main reason for the discriminator updating causing such an increase in people is the fact that someone mentioned it in the discordapp subreddit where it became on of the "hot" posts on it, easily reaching hundreds if not thousands of people.

The actual post: https://www.reddit.com/r/discordapp/comments/147tk7r/discord_competitor_revolt_revolutionises/

In addition are there now more and more posts on that subreddit regarding Revolt. So it only becomes more interesting for people.

**Reply by searinminecraft** [reply]
now lets see if the subreddit now prohibits mention of competitors like revolt

**Comment by SachaTending** [comment]
skill issue

**Comment by tameid** [comment]
one morbillion users..... nor way....
